titus-server-runtime/src/main/java/com/netflix/titus/runtime/endpoint/v3/rest/LoadBalancerResource.java

```diff
@@ -0,0 +1,70 @@
+package com.netflix.titus.runtime.endpoint.v3.rest;
+
+import com.netflix.titus.grpc.protogen.*;
+import com.netflix.titus.runtime.endpoint.common.rest.Responses;
+import com.netflix.titus.runtime.service.LoadBalancerService;
+import io.swagger.annotations.Api;
+import io.swagger.annotations.ApiOperation;
+
+import javax.inject.Inject;
+import javax.inject.Singleton;
+import javax.ws.rs.*;
+import javax.ws.rs.core.*;
+
+@Produces(MediaType.APPLICATION_JSON)
+@Consumes(MediaType.APPLICATION_JSON)
+@Api(tags = "Load Balancing")
+@Path("/v3/loadBalancers")
+@Singleton
+public class LoadBalancerResource {
+    private final LoadBalancerService loadBalancerService;
+
+    @Inject
+    public LoadBalancerResource(LoadBalancerService loadBalancerService) {
+        this.loadBalancerService = loadBalancerService;
+    }
+
+    @GET
+    @ApiOperation("Find the load balancer with the specified ID")
+    @Path("/{jobId}")
+    public Response getJobLoadBalancers(@PathParam("jobId") String jobId) {
```

Comment:
Should the return here be `GetLoadBalancersResult`? Is this not causing a `ClassCastException`?

Reply:
Can there be multiple load balancers associated with a job? Also, this should be an error at runtime as fromSingleValueObservable would not return type `Response`.

Reply:
Thanks @fabiokung,  as noted by @corindwyer below I forgot to write a test which would have caught that.  I've written the test and fixed the bug.